cmd/ipam-controller/app/options/options.go

```diff
@@ -32,6 +32,7 @@ func New() *Options {
 		LeaderElectionNamespace: "kube-system",
 		ConfigMapName:           "nvidia-k8s-ipam-config",
```

Comment:
maybe it is better to remove ConfigMap related options right now? I understand that we need this info for migration logic. Maybe we can keep these values configurable with "secret" env variables to cover edge cases and in all other cases simply use defaults. IMO, by default migrator should search for ConfigMap in the namespace which is set by `--ippools-namespace` cmd line argument and use `nvidia-k8s-ipam-config` as a name.

Reply:
Sure, I will fix it